src/containers/NimbledroidSummaryTable/index.jsx

```diff
@@ -23,22 +23,32 @@ class NimbledroidSummaryTable extends Component {
     showSites: false,
   };
 
+  constructor(props) {
+    super(props);
+    const { nimbledroidData } = this.props;
+    if (nimbledroidData) {
+      this.state = { ...(this.generateData(this.props)) };
+    }
```

Comment:
So this is basically receiving empty data along with other props? Seems a bit odd, but if you need to do it this way it'd be good to add a comment explaining why. :)

Reply:
Pretty much!

It is not for testing purposes. I wonder if I actually had a good reason or pattern or if I'm being plain lazy.

I will change it. Thanks for your review!